commands/jobs.go

```diff
@@ -357,14 +357,17 @@ func handleJobsSubmit(ds *discordgo.Session, ic *discordgo.InteractionCreate) (*
 		Embeds: []*discordgo.MessageEmbed{
```

Comment:
Can we add the ping to the user in the content? discord for some stupid reason doesn't load users if they are not in the cache if their name only appears in an embed.

Reply:
Sorry I'm not familiar with discord APIs, so idk what you mean by that. I guess what you want confirm is `ic.Member.Mention()` and `"<@" + ic.Member.User.ID + ">"` does exactly same thing?

Reply:
yeah, im just saying that in the `&discordgo.MessageSend{` (a few lines up` struct, there's a field for `Content` which is the raw message content that is outside of the embeds. If we make the content the `ic.Member.Mention()` then the discord client will properly load the user instead of just showing `<@923470192437>` for some silly reason